package.json

```diff
@@ -128,11 +128,12 @@
     "snyk": "^1.161.1",
     "source-map-explorer": "1.8.0",
     "tslint-no-circular-imports": "0.6.2",
-    "typescript": "3.4.5"
+    "typescript": "3.5"
   },
   "resolutions": {
     "@types/react": "16.7.13",
-    "js-yaml": "3.13.1"
+    "js-yaml": "3.13.1",
+    "hoist-non-react-statics": "3.3.0"
```

Comment:
@cfcosta what is this line for?

Reply:
```
hoist-non-react-statics had improper typescript typings and had to
be forcefully updated. I did some investigations on the changes for
the versions, nothing seems non-breaking.
```

From the description. But expanding, it is used by react-redux to do **stuff**, and the type definitions for it are broken, it fails compilation when `@types/react-redux` imports types from it.